src/gretel_trainer/models.py

```diff
@@ -16,6 +16,16 @@
 
 
 def determine_best_model(df: pd.DataFrame):
+    """
+    Determine which Gretel model would best suited for generating synthetic data
+    for your dataset. 
+
+    Args:
+        df (pd.DataFrame): Pandas DataFrame containing the data you intend to synthesize.
```

Comment:
I think this should be something like "the data used to train a synthetic model"

Reply:
Updated in a new commit